src/phoenix/experimental/evals/models/openai.py

```diff
@@ -58,6 +44,24 @@ class OpenAIModel(BaseEvalModel):
     """Maximum number of seconds to wait when retrying."""
 
     def __post_init__(self) -> None:
+        self._init_environment()
+        self._init_open_ai()
+
+    def _init_environment(self) -> None:
+        try:
+            import openai
+            from openai import error as oa_err
```

Comment:
Is this one needed? Not possible to import `openai` only and then reference `openai.error.RateLimitError`, for example?

Nit: `openai_error` is more explicit.

Reply:
It does not error per-se but the type checker does not like it. It is the same situation we talked about. It complains when you do `openai.error` but all is good when you do `from openai import error as openai_error`